sunpy/timeseries/timeseriesbase.py

```diff
@@ -26,6 +26,10 @@
 from astropy.table import Table
 from astropy.table import Column
 
+# define and register a new unit, needed for RHESSI
+det = u.def_unit('detector')
+u.add_enabled_units([det])
```

Comment:
This adds a new unit to the unit registry which enables users to use it. It is only defined when they import timeseries. We could consider making a PR to astropy itself to add it.